app-backend/handler.js

```diff
@@ -19,13 +20,13 @@ exports.graphqlHandler = function graphqlHandler(event, context, callback) {
     callback(error, output);
   }
 
-  const handler = server.graphqlLambda({ schema: myGraphQLSchema });
+  const handler = graphqlLambda({ schema: myGraphQLSchema });
   return handler(event, context, callbackFilter);
 };
 
 // for local endpointURL is /graphql and for prod it is /stage/graphql
-exports.graphiqlHandler = server.graphiqlLambda({
```

Comment:
I think we should not remove /graphiql but rather have both /graphiql and playground in this integration.